analyzer/analyzer.go

```diff
@@ -195,10 +194,16 @@ func (a Analyzer) AnalyzeFile(ctx context.Context, wg *sync.WaitGroup, limit *se
 		if !d.Required(strings.TrimLeft(filePath, "/"), info) {
 			continue
 		}
-		b, err := opener()
+		rc, cleaner, err := opener()
 		if err != nil {
 			return xerrors.Errorf("unable to open a file (%s): %w", filePath, err)
 		}
+		defer func() {
+			err := cleaner()
+			if err != nil {
+				log.Logger.Warn("Clean temp directory error: %s", err)
+			}
+		}()
```

Comment:
Seems like this defer will be called many times for the same file and it causes an error possibly. We have to be careful about when we call `cleaner()`.